<ui version="4.0" >
 <class>TorrentDetailsForm</class>
 <widget class="QWidget" name="TorrentDetailsForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>661</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="tabPosition" >
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string comment="Amount of downloaded data" >&lt;b>Total download:&lt;/b></string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2" >
        <widget class="QLabel" name="labelTotalDownload" />
       </item>
       <item row="0" column="3" >
        <widget class="QLabel" name="label_10" >
         <property name="text" >
          <string comment="Amount of uploaded data" >&lt;b>Total upload:&lt;/b></string>
         </property>
        </widget>
       </item>
       <item row="0" column="4" colspan="2" >
        <widget class="QLabel" name="labelTotalUpload" />
       </item>
       <item row="1" column="0" colspan="6" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Completition</string>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>5</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>5</number>
          </property>
          <property name="bottomMargin" >
           <number>7</number>
          </property>
          <item row="0" column="0" >
           <widget class="TorrentProgressWidget" native="1" name="widgetCompletition" >
            <property name="minimumSize" >
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="6" >
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Availability</string>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>5</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>5</number>
          </property>
          <property name="bottomMargin" >
           <number>7</number>
          </property>
          <item row="0" column="0" >
           <widget class="TorrentProgressWidget" native="1" name="widgetAvailability" >
            <property name="minimumSize" >
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" colspan="2" >
        <widget class="QLabel" name="label" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>&lt;b>Tracker:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" >
        <widget class="QLabel" name="labelTracker" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="3" colspan="2" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>&lt;b>Private torrent:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5" >
        <widget class="QLabel" name="labelPrivate" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2" >
        <widget class="QLabel" name="label_2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>&lt;b>Availability:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" >
        <widget class="QLabel" name="labelAvailability" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="3" >
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>&lt;b>Share ratio:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="5" >
        <widget class="QLabel" name="labelRatio" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2" >
        <widget class="QLabel" name="label_3" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>&lt;b>Piece length:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2" >
        <widget class="QLabel" name="labelPieceLength" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="3" colspan="2" >
        <widget class="QLabel" name="label_6" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>&lt;b>Creator:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="5" >
        <widget class="QLineEdit" name="lineCreator" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frame" >
          <bool>false</bool>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2" >
        <widget class="QLabel" name="label_7" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>&lt;b>Creation date:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2" >
        <widget class="QLabel" name="labelCreationDate" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2" >
        <widget class="QLabel" name="label_5" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>&lt;b>Comment:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2" colspan="4" >
        <widget class="QTextBrowser" name="textComment" >
         <property name="openExternalLinks" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>202</width>
           <height>81</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Peers</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QTreeView" name="treePeers" >
         <property name="contextMenuPolicy" >
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights" >
          <bool>true</bool>
         </property>
         <property name="itemsExpandable" >
          <bool>false</bool>
         </property>
         <property name="animated" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Pieces</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QTreeView" name="treePieces" >
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights" >
          <bool>true</bool>
         </property>
         <property name="itemsExpandable" >
          <bool>false</bool>
         </property>
         <property name="animated" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4" >
      <attribute name="title" >
       <string>Files</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QTreeView" name="treeFiles" >
         <property name="contextMenuPolicy" >
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="selectionMode" >
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights" >
          <bool>true</bool>
         </property>
         <property name="itemsExpandable" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionOpenFile" >
   <property name="icon" >
    <iconset resource="../gfx/resources.qrc" >:/menu/run.png</iconset>
   </property>
   <property name="text" >
    <string>Open</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TorrentProgressWidget</class>
   <extends>QWidget</extends>
   <header>engines/TorrentProgressWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../gfx/resources.qrc" />
 </resources>
 <connections/>
</ui>
